modules/atomic-widgets/styles/style-schema.php

```diff
@@ -232,17 +232,22 @@ private static function get_alignment_props() {
 				'space-evenly',
 				'stretch',
 			]),
-			'align-items' => String_Prop_Type::make()->enum([
+			'align-content' => String_Prop_Type::make()->enum([
 				'normal',
```

Comment:
I see that here we have a normal value but in the packages it doesn't exists.
Do we need it?